src/nevermined/Nevermined.ts

```diff
@@ -49,7 +35,7 @@ export class Nevermined extends Instantiable {
      * @param config - Nevermined instance configuration.
      * @returns A {@link Nevermined} instance
      */
-    public static async getInstance(config: Config): Promise<Nevermined> {
+    public static async getInstance(config: NeverminedOptions): Promise<Nevermined> {
```

Comment:
can we replace this with a normal constructor?

Reply:
I don't think that provides value for this class and would limit some future improvements like introduce singletons